Content.Shared/RCD/Systems/RCDSystem.cs

```diff
@@ -446,11 +446,6 @@ private bool IsConstructionLocationValid(EntityUid uid, RCDComponent component,
                     if (!fixture.Hard || fixture.CollisionLayer <= 0 || (fixture.CollisionLayer & (int) prototype.CollisionMask) == 0)
                         continue;
 
-                    // Continue if our custom collision bounds are not intersected
-                    if (prototype.CollisionPolygon != null &&
-                        !DoesCustomBoundsIntersectWithFixture(prototype.CollisionPolygon, component.ConstructionTransform, ent, fixture))
-                        continue;
-
```

Comment:
Might be better off just iterating fixtures directly and querying, or using the engine method (I forgor which one it is). Worst case add an entitylookup that checks each fixture individually?